### DISCUSSION TITLE
“Answer” and “Suggest an Answer” phrasing—should be “Reply” or “Post”

### DISCUSSION THREAD

**Discussion by hughlilly** [community/community#6179] [Discussions]
Answers are designated as such only by a Discussion’s creator, right? If that’s the case I think it’s presumptuous to imply that other users, when adding posts in response to that Discussion, are suggesting the final Answer. They are merely adding to the discussion. Sure, their post _may be selected as the Answer_,  but they can’t know that for sure at the time.

I think the phrasing should change. On desktop this would mean having “Post” in place of “Answer” as the text for the submit button on the reply form, and the greyed-out “Suggest an answer” prompt text could instead read “Add a reply”. On mobile, you could have the floating button say “Post” instead of “Suggest Answer”.

Thanks
Hugh

**Comment by laymonage** [comment]
Related: #5230.
